src/DGEList.py

```diff
@@ -25,11 +24,11 @@ class DGEList(object):
         genes: Array of gene names, same length as nrow(counts).
         norm_factors: Weighting factors for each sample.
         group: ...
-        remove_zeroes: To remove genes with zero counts for all samples.
+        to_remove_zeroes: To remove genes with zero counts for all samples.
 
     Examples:
         >>> import gzip
-        >>> from edgePy.io import get_dataset_path
+        >>> from src.data_import.data_import import get_dataset_path
```

Comment:
Why can't we have submodule `io`? It makes more sense, intuitively, since we describe input/output of data there.

It was working before FYI. Check out other libraries that do it:

http://scikit-bio.org/docs/0.4.0/io.html

Reply:
io clashes with the built in "io" module.  You shouldn't name modules after existing built in packages.